src/core/proj/qgscoordinatereferencesystem.cpp

```diff
@@ -2354,8 +2354,8 @@ int QgsCoordinateReferenceSystem::syncDatabase()
 
   PROJ_STRING_LIST authorities = proj_get_authorities_from_database( pjContext );
 
-  int nextSrsId = 63321;
-  int nextSrId = 520003321;
+  int nextSrsId = 63560;
+  int nextSrId = 520003560;
```

Comment:
Hi @nyalldawson, it seems there is an issue here.
The srsid 63560 - srid 520003560 ("GSK-2011 / Gauss-Kruger zone 4" EPSG:20904) is already present in the srs6.db updated with this same PR.
See https://lists.osgeo.org/pipermail/qgis-user/2021-December/050357.html
Probably triggered by https://github.com/OSGeo/PROJ/pull/2943

Reply:
See https://github.com/qgis/QGIS/pull/46492